application/event_processor/tests/test_event_processor.py

```diff
@@ -372,3 +381,41 @@ def test_update_changes_address_to_change_request_if_not_equal_not_equal():
     actual_changes = update_changes_with_address(changes, "address", dos_address, nhs_uk_entity)
     # Assert
     assert expected_changes == actual_changes, f"Should return {changes} dict, actually: {actual_changes}"
+
+
+@patch.object(Logger, "info")
```

Comment:
Great unit test, if you shift the function to another file can a test file be created with this unit test in as well

Reply:
Sure . agree